docs/mddocs/Quickstart/ollama_portablze_zip_quickstart.md

```diff
@@ -79,4 +84,53 @@ To increase the context length, you could set environment variable `IPEX_LLM_NUM
 - Start Ollama serve through `start-ollama.bat`
 
 > [!TIP]
-> `IPEX_LLM_NUM_CTX` has a higher priority than the `num_ctx` settings in a models' `Modelfile`.
\ No newline at end of file
+> `IPEX_LLM_NUM_CTX` has a higher priority than the `num_ctx` settings in a models' `Modelfile`.
+
+----------------------------
+
+## Linux Quickstart
+
+### Prerequisites
+
+Check your GPU driver version, and update it if needed:
+
+- For client GPU, like A-series, B-series and integrated GPU, we recommend following [Intel client GPU driver installing guide](https://dgpu-docs.intel.com/driver/client/overview.html) to install your GPU driver.
+- For Data Center GPU, like Max series and FLEX series, we recommend following [Intel Data Center GPU driver installing guide](https://dgpu-docs.intel.com/driver/installation.html) to install your GPU driver.
+
+### Step 1: Download and Extract
+
+Download IPEX-LLM Ollama portable tgz from the [link](https://github.com/intel/ipex-llm/releases/tag/v2.2.0-nightly).
+
+Then open a terminal, extract the tgz file to a folder.
+```bash
+cd PATH/TO/DOWNLOADED/TGZ
+tar xvf [Downloaded tgz file]
+```
+
+### Step 2: Start Ollama Serve
+
+Enter the extracted folder, and run `start-ollama.sh` to start Ollama service.  
+
+[Optional]For multi-gpus user, please edit `start-ollama.sh` in the extracted folder and change `ONEAPI_DEVICE_SELECTOR` according to your configuration.
```

Comment:
Move this to a note or tip; BTW, is this also applicable to Windows?

Reply:
Yes. It is also applicable to Windows.

I will raise a further PR to move `Tips & Troubleshooting` section across Windows & Linux, to include ctx, model source, and select GPU, after supporting model source on Linux.